foreman_os_default_template.py

```diff
@@ -151,10 +159,12 @@ def main():
 
     module = AnsibleModule(
         argument_spec=dict(
-            operatingsystem=dict(type='str', required=True),
             config_template=dict(type='str', required=True),
-            template_kind=dict(type='str', required=True),
+            major=dict(type='int', required=True),
```

Comment:
Hui, required would make playbooks not passing the parameter invalid. Is it really necessary?